Puppetfile

```diff
@@ -16,9 +16,11 @@ mod 'puppetlabs/mysql',      '3.0.0'
 mod 'puppetlabs/postgresql', '4.1.0'
 mod 'puppetlabs/mongodb',    '0.8.0'
 mod 'puppetlabs/rabbitmq',   '4.1.0'
+mod 'puppetlabs-gcc',        '0.3.0'
 mod 'jfryman/nginx',         '0.2.1'
 mod 'mayflower/php',         '3.4.2'
 mod 'mayflower/hhvm',        '0.1.2'
 mod 'willdurand/nodejs',     '1.8.3'
 mod 'maestrodev/wget',       '1.5.6'
 mod 'nanliu/staging',        '1.0.2'
+mod 'thomasvandoren/redis',  '0.10.0'
```

Comment:
This module compiles redis from source, that's why the gcc module is needed. We prefer installing software via packages.

Why is this needed?